scripts/preprocess_sentences.py

```diff
@@ -63,8 +186,14 @@ def rap_word_expansion(sentence):
     :param sentence: [String] Sentence to expand.
     :return: Expanded sentence.
     """
+    sentence = expand_contractions(sentence)
     return sentence.replace("in' ", "ing ")
 
+def expand_contractions(text, c_re=c_re):
```

Comment:
Please add a docstring for this function (mainly just to explain what c_re is and what is happening in the nested function below).